.rubocop.yml

```diff
@@ -1,523 +1,4 @@
-Style/DoubleCopDisableDirective:
-  Enabled: false
-Style/EmptyBlockParameter:
-  Enabled: false
-Style/EmptyLambdaParameter:
-  Enabled: false
-Style/Encoding:
-  Enabled: false
-Style/EvalWithLocation:
-  Enabled: false
-Style/ExpandPathArguments:
-  Enabled: false
-Style/ExplicitBlockArgument:
-  Enabled: false
-Style/ExponentialNotation:
-  Enabled: false
-Style/FloatDivision:
-  Enabled: false
-Style/FrozenStringLiteralComment:
-  Enabled: false
-Style/GlobalStdStream:
-  Enabled: false
-Style/HashAsLastArrayItem:
-  Enabled: false
-Style/HashLikeCase:
-  Enabled: false
-Style/HashTransformKeys:
-  Enabled: false
-Style/HashTransformValues:
-  Enabled: false
-Style/IfUnlessModifier:
-  Enabled: false
-Style/KeywordParametersOrder:
-  Enabled: false
-Style/MinMax:
-  Enabled: false
-Style/MixinUsage:
-  Enabled: false
-Style/MultilineWhenThen:
-  Enabled: false
-Style/NegatedUnless:
-  Enabled: false
-Style/NumericPredicate:
-  Enabled: false
-Style/OptionalBooleanParameter:
-  Enabled: false
-Style/OrAssignment:
-  Enabled: false
-Style/RandomWithOffset:
-  Enabled: false
-Style/RedundantAssignment:
-  Enabled: false
-Style/RedundantCondition:
-  Enabled: false
-Style/RedundantConditional:
-  Enabled: false
-Style/RedundantFetchBlock:
-  Enabled: false
-Style/RedundantFileExtensionInRequire:
-  Enabled: false
-Style/RedundantRegexpCharacterClass:
-  Enabled: false
-Style/RedundantRegexpEscape:
-  Enabled: false
-Style/RedundantSelfAssignment:
-  Enabled: false
-Style/RedundantSort:
-  Enabled: false
-Style/RescueStandardError:
-  Enabled: false
-Style/SingleArgumentDig:
-  Enabled: false
-Style/SlicingWithRange:
-  Enabled: false
-Style/SoleNestedConditional:
-  Enabled: false
-Style/StderrPuts:
-  Enabled: false
-Style/StringConcatenation:
-  Enabled: false
-Style/Strip:
-  Enabled: false
-Style/SymbolProc:
-  Enabled: false
-Style/TrailingBodyOnClass:
-  Enabled: false
-Style/TrailingBodyOnMethodDefinition:
-  Enabled: false
-Style/TrailingBodyOnModule:
-  Enabled: false
-Style/TrailingCommaInHashLiteral:
-  Enabled: false
-Style/TrailingMethodEndStatement:
-  Enabled: false
-Style/UnpackFirst:
-  Enabled: false
-Lint/DuplicateBranch:
-  Enabled: false
-Lint/DuplicateRegexpCharacterClassElement:
-  Enabled: false
-Lint/EmptyBlock:
-  Enabled: false
-Lint/EmptyClass:
-  Enabled: false
-Lint/NoReturnInBeginEndBlocks:
-  Enabled: false
-Lint/ToEnumArguments:
-  Enabled: false
-Lint/UnexpectedBlockArity:
-  Enabled: false
-Lint/UnmodifiedReduceAccumulator:
-  Enabled: false
-Performance/CollectionLiteralInLoop:
-  Enabled: false
-Style/ArgumentsForwarding:
-  Enabled: false
-Style/CollectionCompact:
-  Enabled: false
-Style/DocumentDynamicEvalDefinition:
-  Enabled: false
-Style/NegatedIfElseCondition:
-  Enabled: false
-Style/NilLambda:
-  Enabled: false
-Style/RedundantArgument:
-  Enabled: false
-Style/SwapValues:
-  Enabled: false
-RSpec/FilePath:
-  Enabled: false
\ No newline at end of file
+inherit_gem:
+  puppet-lint: .rubocop.yml
+
```

Comment:
we may need to add `require: MultilineBlocks` at the top as this is where tests are failing